src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1262,6 +1387,12 @@ namespace Microsoft.Management.Deployment
             // Get download progress
             Windows.Foundation.IAsyncOperationWithProgress<DownloadResult, PackageDownloadProgress> GetDownloadProgress(CatalogPackage package, PackageCatalogInfo catalogInfo);
         }
+
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 11)]
+        {
+            // Repair the specified package
+            Windows.Foundation.IAsyncOperationWithProgress<RepairResult, RepairProgress> RepairPackageAsync(CatalogPackage package, RepairOptions options);
+        }

```

Comment:
Missing GetRepairProgress

Reply:
During the API review process, we decided against including GetRepairProgress as we couldn't identify a scenario where another client would need to track the asynchronous operation for repair.